Source/Core/Core/Slippi/SlippiMatchmaking.cpp

```diff
@@ -17,7 +17,7 @@ class MmMessageType
 std::string MmMessageType::CREATE_TICKET = "create-ticket";
 std::string MmMessageType::CREATE_TICKET_RESP = "create-ticket-resp";
 std::string MmMessageType::GET_TICKET_RESP = "get-ticket-resp";
-extern std::atomic<bool> connectionsReset = true;
+extern std::atomic<bool> connectionsReset{true};
```

Comment:
you can just get rid of this if you want - https://github.com/davecusatis/Ishiiruka/commit/267035d3ee1767bb2698dee5901f6074d8588ade#diff-1546f909ffabc1d7ab254bf78eec58d371089b545893619dc7646f35b7f1344fL20

i implemented these same changes more or less to get a working mac build